app/src/components/main/image/image.jsx

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'react';
```

Comment:
I think will be fine to add fallback image and loading state to that component.

Reply:
fixed